components/skeleton/skeleton-mixin.js

```diff
@@ -155,7 +156,8 @@ export const SkeletonMixin = dedupeMixin(superclass => class extends RtlMixin(su
 			 * Renders the input as a [skeleton loader](https://github.com/BrightspaceUI/core/tree/main/components/skeleton)
 			 * @type {boolean}
 			 */
-			skeleton: { reflect: true, type: Boolean  }
+			skeleton: { reflect: true, type: Boolean  },
```

Comment:
So it doesn't get forgotten once this goes out of Draft: this needs documentation.